pyproject.toml

```diff
@@ -176,13 +176,12 @@ debian = [
     "silver-platter[debian]",
 ]
 
-# Google Cloud Platform
-gcp = [
+# Logging (./janitor.conf.example:logs_location)
+cloud = [
+    # GCP - Google Cloud Platform
```

Comment:
Please keep these separate - e.g. the Debian janitor doesn't need the S3 support and both it and the Google cloud modules pull in a lot of extra dependencies.